src/components/ButtonPanel.jsx

```diff
@@ -2,39 +2,30 @@ import React from 'react';
 import Button from './Button';
 
 function ButtonPanel() {
-  return (
-    <div>
-      <div id="group1">
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
-        <Button name="÷" />
-      </div>
-      <div id="group2">
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
-        <Button name="X" />
-      </div>
-      <div id="group3">
-        <Button name="4" />
-        <Button name="5" />
-        <Button name="6" />
-        <Button name="-" />
-      </div>
-      <div id="group4">
-        <Button name="1" />
-        <Button name="2" />
-        <Button name="3" />
-        <Button name="+" />
-      </div>
-      <div id="group5">
-        <Button name="0" />
-        <Button name="." />
-        <Button name="=" />
-      </div>
+  const symbolArray = [
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+
+  const listedButton = (btnGroup => btnGroup.map(button => {
+    let buttonComponent;
+    if (['÷', 'X', '-', '+'].includes(button)) {
```

Comment:
we are currently having this 👇 ;
![image](https://user-images.githubusercontent.com/31897434/95142487-b84ed000-076b-11eb-8882-300c212127aa.png)

`=` should be in orange color too.

we need to add `=` this array. includes right? 

And also, let's make use of lowercase `x` the current sign is looking way bigger than the rest of the signs.

Reply:
Yes, you're right. I'm on it.